Update/docs (#543)
desmos-labs/desmos: docs/fullnode/setup.md

db_backend = "rocksdb"
```

## 5. (Optional) Snapshots
Snapshots allow nodes to use state sync and quickly sync up. While it is optional, we extremely recommend turning this feature on to keep the chain stable.

> We will assume you have set pruning to `default`. If that is not the case, please adjust accordingly

Reviewer: ```suggestion
We assume you have set pruning to `default`. If that is not the case, please adjust accordingly
```

> In order to turn this on, please to inside the `app.toml` file and change the following:

Reviewer: ```suggestion
In order to turn this on, please open the `app.toml` file inside `.desmos` directory and change the following:
```

- port `9090` to expose the [gRPC](https://grpc.io/) service that allows clients to query the chain state
- port `1317` to expose the REST APIs service

> While opening any ports are optional, it is beneficial to the whole chain if

Reviewer: ```suggestion
While opening any ports are optional, it is beneficial to the whole network if
```

> you open port `26656`. This would allow new nodes to connect to you as a peer, making their sync faster and the connection more reliable.

Reviewer: ```suggestion
you open port `26656`. This would allow new nodes to connect to you as a peer, making them sync faster and the connections more reliable.
```
